### DISCUSSION TITLE
How to add current date in yaml "date" in Julia

### DISCUSSION THREAD

**Discussion by awojtas-r** [quarto-dev/quarto-cli#11477] [Q&A] [answered]
### Description

I'm trying to add actual date in yaml using Julia `"date: "Created on: `{julia} Dates.today()`"` but that prints 'invalid date' on html output, which is weird because using R "date: `"Created on: `r format(Sys.time(), '%e %B, %Y')`"` works fine, any advice how to do it in Julia?

**Comment by mcanouil** [comment, marked as answer]
There is no need to use any languages to compute the date, it's even not recommended.

https://quarto.org/docs/reference/dates.html

**Reply by awojtas-r** [reply]
thank you!
